administrator-manual/en/webtop5.rst

```diff
@@ -423,6 +423,12 @@ A precompiled wizard will open:
    Remember to fill in the User name and Password fields related to access to the Nextcloud resource,
    otherwise it will not be possible to use the public link to the shared files
 
+.. note::
+
+   If NextCloud is configured with a custom virtual host (eg **mycloud.domain.xxx**) the :guilabel:`Path` must be changed in this way **"/remote.php/webdav"**
+   (eliminating **/nextcloud**) and in the :guilabel:`Host` name it is necessary to enter the name of the custom virtual host.
```

Comment:
If Nextcloud has been configured with a custom virtual host (eg ``nextcloud.mydomain.com``) the :guilabel:`Path` must be changed to from `/nextcloud/remote.php/webdav` to `/remote.php/webdav`, please note that `/nextcloud` prefix has been removed. Also make sure to enter the name of the custom virtual host inside the :guilabel:`Host` (eg. ``nextcloud.mydomain.com``).